front/src/common/api/generatedEditoastApi.ts

```diff
@@ -2905,6 +2931,7 @@ export type RollingStock = {
   effort_curves: EffortCurves;
   electrical_power_startup_time: number | null;
   energy_sources: EnergySource[];
+  etcs_brake_params: EtcsBrakeParams | null;
```

Comment:
Hm, why is this field not optional (`?`) while it's optional in `RollingStockMetadata`?